### PR TITLE
added Docker Trusted Registry information

### PR DESCRIPTION
**Pull request by abhijitkini**

### CODE REVIEW

**Comment by abhijitkini** [comment]
https://github.com/Shippable/docsv2/issues/339

File: sources/ci_configure.md [hunk 1]
```diff
@@ -382,6 +382,29 @@ integrations:
...
+    	- docker push ship-dtr.in/sdemo/sample_app:$BRANCH.$BUILD_NUMBER
+
+integrations:
+	hub:
+	  - integrationName: "ship-ddc-dtr"
+        type: "Docker Trusted Registry"
+        branches:
+          only:
+            - master
+```
```
Comment (manishas) [inline sources/ci_configure.md]: type and branches need to be aligned with integrationName. only and master will also need to be indented correctly after the indent for branches is fixed.

File: sources/ci_configure.md [hunk 2]
```diff
@@ -382,6 +382,29 @@ integrations:
...
+
+In the above example: `ship-dtr-in/sdemo/sample_app` is the Docker Trusted Registry URL and `$BUILD.$BUILD_NUMBER` is the tag.
+
```
Comment (manishas) [inline sources/ci_configure.md]: in the yml, it's ship-dtr.in so we should keep it consistent. Also, the trusted registry URL is just the ship-dtr.in, and the ship-dtr-in/sdemo/sample_app is probably called something else... pls look at docker docs to see how it is explained - https://docs.docker.com/docker-trusted-registry/repos-and-images/push-and-pull-images/

we're pushing the image with the tag `$BUILD.$BUILD_NUMBER`.

File: sources/ci_configure.md [hunk 3]
```diff
@@ -382,6 +382,29 @@ integrations:
...
+	hub:
+      - integrationName: "ship-ddc-dtr"
+        type: "Docker Trusted Registry"
+        branches:
+          only:
+            - master
+```
+
+In the above example: `ship-dtr.in` is the Docker Trusted Registry URL, `sdemo` is the username. The image tag is set to `sample_app:$BUILD.$BUILD_NUMBER`.
+
```
Comment (manishas) [inline sources/ci_configure.md]: Isn't sample_app the image name?

File: sources/ci_configure.md [hunk 4]
```diff
@@ -382,6 +382,29 @@ integrations:
...
+```
+build:
+  	post_ci:
+    	#Commit the container only if you want all the artifacts from the CI step
+    	- docker commit $SHIPPABLE_CONTAINER_NAME ship-dtr.in/sdemo/sample_app:$BRANCH.$BUILD_NUMBER .
+    	- docker push ship-dtr.in/sdemo/sample_app:$BRANCH.$BUILD_NUMBER
+
+integrations:
+	hub:
+      - integrationName: "ship-ddc-dtr"
```
Comment (manishas) [inline sources/ci_configure.md]: hub and post_ci are mis-indented. please review your PR before sending it....

**Pull request merged**